qa/config/platforms.json

```diff
@@ -0,0 +1,16 @@
+{ 
+  "latest": "5.0.0-alpha2",
+  "platforms" : {
+    "ubuntu-1204": { "box": "elastic/ubuntu-12.04-x86_64", "type": "debian" },
+    "ubuntu-1404": { "box": "elastic/ubuntu-14.04-x86_64", "type": "debian", "specific": true },
+    "ubuntu-1504": { "box": "elastic/ubuntu-15.04-x86_64", "type": "debian" },
```

Comment:
15.04 is deprecated now.  We should probably make an image that uses the new LTS 16.04, as that will be used by many.